bin/lint.ts

```diff
@@ -0,0 +1,29 @@
+import { readdirSync } from 'fs';
+import { join } from 'path';
+
+const MAX_PARALLEL = Number(process.env.GAPI_MAX_PARALLEL) || 1;
```

Comment:
Would be awesome to add [dotenv](https://www.npmjs.com/package/dotenv) support for running linting locally also in parallel.

Reply:
Yeah, this will also work:

```
$ GAPI_MAX_PARALLEL=2 npm run lint
```

Will that suffice for now?

Reply:
Sure, add this snippet to the `README.md` then, please